frontend/src/store/modules/members.js

```diff
@@ -41,6 +41,11 @@ const actions = {
     const namespace = rootState.namespace
     const res = await deleteMember({ namespace, name })
     commit('RECEIVE', res.data)
+  },
+  async rotateServiceAccountSecret ({ commit, rootState }, name) {
+    const namespace = rootState.namespace
+    const res = await deleteServiceAccountSecret({ namespace, name })
+    commit('RECEIVE', res.data)
```

Comment:
We do not change the list

Reply:
I removed the commit, you are right ... same as above